packages/test/test-end-to-end-tests/src/test/summarizeRestart.spec.ts

```diff
@@ -0,0 +1,203 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { strict as assert } from "assert";
+import { ITestDataObject, describeNoCompat, itExpects } from "@fluid-internal/test-version-utils";
+import { IContainer } from "@fluidframework/container-definitions";
+import {
+	ITestContainerConfig,
+	ITestObjectProvider,
+	createSummarizer,
+	mockConfigProvider,
+	summarizeNow,
+} from "@fluidframework/test-utils";
+import { requestFluidObject } from "@fluidframework/runtime-utils";
+
+describeNoCompat("Summarizer closes instead of refreshing", (getTestObjectProvider) => {
+	const settings = {};
+	const testContainerConfig: ITestContainerConfig = {
+		runtimeOptions: {
+			summaryOptions: {
+				summaryConfigOverrides: { state: "disabled" },
+			},
+		},
+		loaderProps: { configProvider: mockConfigProvider(settings) },
+	};
+
+	let provider: ITestObjectProvider;
+	const createContainer = async (): Promise<IContainer> => {
+		return provider.makeTestContainer(testContainerConfig);
+	};
+
+	beforeEach(async () => {
+		provider = getTestObjectProvider({ syncSummarizer: true });
+		settings["Fluid.ContainerRuntime.Test.SummarizationRecoveryMethod"] = "restart";
+	});
+
+	itExpects(
+		"Closes the summarizing client instead of refreshing",
+		[
+			{
+				eventName: "fluid:telemetry:Container:ContainerClose",
+				error: "Restarting summarizer instead of refreshing",
+			},
+		],
+		async () => {
+			const container = await createContainer();
+			const { container: summarizingContainer, summarizer } = await createSummarizer(
+				provider,
+				container,
+				undefined,
+				undefined,
+				mockConfigProvider(settings),
+			);
+
+			const dataStore = await requestFluidObject<ITestDataObject>(container, "default");
```

Comment:
These 2 lines seem unncesary. Same comment for the other tests. It's fine to have them; I am just saying you don't need ops to do summarizeOnDemand :)